tools/RAiDER/cli/raider.py

```diff
@@ -305,8 +305,18 @@ def calcDelays(iargs=None):
 
         # nearest weather model time via 'none' is specified
         # When interp_method is 'none' only 1 weather model file and one relevant time
-        elif (interp_method == 'none') and (len(wfiles) == 1) and (len(times) == 1):
-            weather_model_file = wfiles[0]
+        elif len(wfiles) == 1 and len(times) == 1:
```

Comment:
Why do you remove logic related to `interp_method`? If this is specifically for `center_time` please include this otherwise there is a risk this could effect other workflows.